new_project/examples/btc_eth/package.json

```diff
@@ -7,7 +7,7 @@
         "check": "tsc && tslint -p . && prettier --check '**/*.{ts,json,yml}'",
         "swap": "ts-node ./src/index.ts",
         "fix": "tslint -p . --fix && prettier --write '**/*.{ts,js,json,yml}'",
-        "start-env": "create-comit-app start-env"
```

Comment:
I am not (yet) sure if I like that :thinking: 

It looks very magical to me because the link between `create-comit-app` and `start-env` is now completely missing. It is an old discussion but what if we just follow the create-react-app path and provide a `comit-scripts` binary.

I didn't realize it before but it looks like we can provide several NPM "binaries" as part of one package? That is cool! It would be cool if this would read `comit-scripts start-env`

Reply:
I see. I think we roll with it (maybe I'll choose a better name) and restructure to comit-scripts at a later stage as what you want would most likely mean we publish 2 npm packages: `create-comit-app` and `comit-scripts`. Does this mean they both make you download create-comit-app binary ?

Reply:
Ok I am going to try something :)

Reply:
See description, I did what I think you expect.